squire_lib/src/tournament.rs

```diff
@@ -918,6 +919,60 @@ impl Tournament {
         }
         digest
     }
+
+    /// Generates a round slip in HTML
+    pub fn round_slips_html(&self, css: &str) -> String {
+        let mut ret: String = "<!DOCTPYE HTML>".to_string();
+
+        let _ = write!(ret, "<html lang=\"en\">");
+        let _ = write!(ret, "<head><style>");
+        let _ = write!(ret, "{}", &css);
```

Comment:
Nit pick: while not as smart as, say, Python's f-string formating, Rust's format strings can do simple things like:
``` suggestion
        let _ = write!(ret, "{css}");
```

Reply:
I got errors when I tried this for some reason. I am pretty sure it is because of the borrow needed or something

Reply:
Hmm, that works for me. Maybe you had `"{&css}"`? The format string only accepts basic variable binding, nothing more than that. Unfortunately, even `"{self.name}"` doesn't work.